next-on-netlify.js

```diff
@@ -1,17 +1,28 @@
 #!/usr/bin/env node
 const { program } = require("commander");
 
-program
-  .option(
-    "--max-log-lines [number]",
-    "lines of build output to show for each section",
-    50
-  )
-  .parse(process.argv);
-
 const nextOnNetlify = require("./index");
 const { logTitle } = require("./lib/helpers/logger");
 
-nextOnNetlify();
+program.option(
+  "--max-log-lines [number]",
+  "lines of build output to show for each section",
+  50
+);
+
+program
+  .command("watch")
+  .description("re-runs next-on-netlify on changes")
+  .action(() => {
+    nextOnNetlify({ watch: true });
+  });
+
+program
+  .command("build", { isDefault: true })
+  .description("runs next-on-netlify")
+  .action(() => {
+    nextOnNetlify();
+    logTitle("✅ Success! All done!");
```

Comment:
Should this success log line be moved to after:
https://github.com/netlify/next-on-netlify/blob/6e5c09dd2fe2b6ebc0bdf392869fffa18876c610/index.js#L44

Reply:
doesnt matter too much because the tracking's invisible to the user but sure :)